README.md

```diff
@@ -1,38 +1,71 @@
-Image Processing SciKit
-=======================
+# scikit-image: Image processing in Python
 
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/scikit-image/scikit-image?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 [![Coverage Status](https://img.shields.io/coveralls/scikit-image/scikit-image.svg)](https://coveralls.io/r/scikit-image/scikit-image?branch=master)
 
-Website (including documentation)
----------------------------------
-http://scikit-image.org/
+- **Website (including documentation):** [http://scikit-image.org/](http://scikit-image.org)
+- **Mailing list:** [http://groups.google.com/group/scikit-image](http://groups.google.com/group/scikit-image)
+- **Source:** [https://github.com/scikit-image/scikit-image](https://github.com/scikit-image/scikit-image)
 
-Mailing List
-------------
-http://groups.google.com/group/scikit-image
+## Installation from binaries
 
-Source
-------
-https://github.com/scikit-image/scikit-image
+- **Debian/Ubuntu:** ``sudo apt-get install python-skimage``
+- **OSX:** ``pip install scikit-image``
+- **Anaconda:** ``conda install scikit-image
+- **Windows:** Download [Windows binaries](http://www.lfd.uci.edu/~gohlke/pythonlibs/#scikits.image)
 
-Installation from source
-------------------------
-Refer to DEPENDS.txt for a list of dependencies.
+Also see
+[http://scikit-image.org/docs/dev/install.html](http://scikit-image.org/docs/dev/install.html)
```

Comment:
Maybe name the license explicitly (New BSD / 3-clause BSD)? That way people familiar with it don't need to read through with a fine toothed comb.